### DISCUSSION TITLE
Problem in Upgrading Prisma version from 4.10 to 5.2 in Win 11

### DISCUSSION THREAD

**Discussion by Helios15** [prisma/prisma#21031] [Q&A]
### Question

I want to use Prisma 5.2.0 over my old version 4.10. But when i run " npx prisma generate", i got an issue : "Downloading Prisma engines for Node-API for windows [=========           ] 45%Error: EPERM: operation not permitted, unlink \node_modules\@prisma\engines\schema-engine-windows.exe".  By the way, i can't see my prisma version using prisma -v.
Something wrong with my schema-engine-window.exe. How can i fix it ?

### Information about Prisma Schema, Client Queries and Environment (optional)

- My Schema: 
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Clothes {
  id          Int       @id() @default(autoincrement())
  createdDate DateTime  @default(now()) @map("created_date") @db.Timestamp(6)
  teamId      Int       @map("team_id")
  position    Positions
  url         String    @db.VarChar
  clubName    String    @map("club_name") @db.VarChar
  test        Json[]    @default([]) @db.Json

  @@map("clothes")
}
```

**Comment by ludralph** [comment]
Hi @Helios15 👋 

Can you please delete your `node_modules` folder and try running `npx prisma generate` again to see if it resolves the issue for you?

**Comment by janpio** [comment]
You still seem to have a process running that is blocking the Query Engine from being deleted. Windows does not allow replacing the file while it is being used.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
